src/app/workflow/dag/dag.component.ts

```diff
@@ -70,6 +71,7 @@ export class DagComponent extends EntryTab implements OnInit, OnChanges, AfterVi
   public refreshCounter = 1;
   public dagLoading$: Observable<boolean>;
   public wdlViewerResult$: Observable<boolean>;
+  public isPublic: Boolean = false;
```

Comment:
Should be lower case `boolean`, not `Boolean`: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
Done